### PR TITLE
Hashの値を省略し警告を解消

### PR DESCRIPTION
**Pull request by 88-99**
## Issue
[slim-lint 対応-1 #6994](https://github.com/fjordllc/bootcamp/issues/6994)

## 概要
上記Issueに記載されている対象のファイルを修正しました。

## 変更確認方法
1. `chore/fix-slim-lint-issues-6994`をローカルに取り込む
2. `config/slim_lint.yml`に追加されている除外ルール（`- Style/HashSyntax`）を削除。
3. `bundle exec slim-lint app/views -c config/slim_lint.yml`を実行し、警告が解消されているか確認

## Screenshot
修正前後のbundle exec slim-lint app/views -c config/slim_lint.ymlの実行結果を記載します。

### 変更前
app/views/admin/users/_nav.html.slim:6 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_sort_column.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_sort_column.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_sort_column.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_table.html.slim:13 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_table.html.slim:18 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_table.html.slim:21 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_table.html.slim:8 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/application/_global_nav.slim:14 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/application/_page_tabs.html.slim:11 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/application/_page_tabs.html.slim:13 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/application/_page_tabs.html.slim:8 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/comments/_user_icons.html.slim:2 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/companies/_company.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/companies/_profile.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/companies/users/_users_tabs.html.slim:6 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/events/_event.html.slim:47 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/events/_event.html.slim:59 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/generations/_users_tabs.html.slim:9 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/home/_collegue_trainee.html.slim:4 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/home/_job_seeking_users.html.slim:11 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/home/_job_seeking_users.html.slim:39 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_first_page.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_last_page.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_next_page.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_page.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_page.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_page.html.slim:9 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_page.html.slim:9 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_prev_page.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/notifications/_tabs.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/pages/unauthorized_show.slim:7 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/pages/index.html.slim:5 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/products/index.html.slim:8 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/questions/index.html.slim:5 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/reports/index.html.slim:5 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/show.html.slim:6 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/unauthorized_show.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/index.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/self_assigned/index.html.slim:16 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/show.html.slim:8 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/unassigned/index.html.slim:14 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/unchecked/index.html.slim:22 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/questions/unauthorized_show.slim:7 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/regular_events/_regular_event.html.slim:45 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/regular_events/_regular_event.html.slim:60 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/reports/_report_body.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/reports/show.html.slim:72 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/shared/_modal_learning_completion.html.slim:2 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/survey_questions/_form.html.slim:50 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/survey_questions/_form.html.slim:51 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/survey_questions/_form.html.slim:52 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/talks/_nav.html.slim:6 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/talks/show.html.slim:86 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:101 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:103 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:105 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:11 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:114 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:114 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:124 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:16 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:16 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:17 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:17 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:18 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:18 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:19 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:19 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:23 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:24 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:28 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:28 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:55 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:56 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:57 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:58 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:65 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:74 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:76 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:78 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:86 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:86 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:99 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_inactive_users.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_password_change.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_profile.html.slim:36 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/form/_description.html.slim:11 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/practices/_completed_practices.html.slim:5 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/practices/_completed_practices.html.slim:7 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/welcome/_administrator.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.

### 変更後
app/views/admin/users/_table.html.slim:13 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_table.html.slim:18 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_table.html.slim:21 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/admin/users/_table.html.slim:8 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/comments/_user_icons.html.slim:2 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/companies/_company.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/companies/_profile.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/companies/users/_users_tabs.html.slim:6 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/events/_event.html.slim:47 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/events/_event.html.slim:59 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/generations/_users_tabs.html.slim:9 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/home/_collegue_trainee.html.slim:4 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/home/_job_seeking_users.html.slim:11 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/home/_job_seeking_users.html.slim:39 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_first_page.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_last_page.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_next_page.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_page.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_page.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_page.html.slim:9 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_page.html.slim:9 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/kaminari/_prev_page.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/notifications/_tabs.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/pages/unauthorized_show.slim:7 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/pages/index.html.slim:5 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/products/index.html.slim:8 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/questions/index.html.slim:5 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/reports/index.html.slim:5 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/show.html.slim:6 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/practices/unauthorized_show.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/index.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/self_assigned/index.html.slim:16 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/show.html.slim:8 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/unassigned/index.html.slim:14 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/products/unchecked/index.html.slim:22 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/questions/unauthorized_show.slim:7 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/regular_events/_regular_event.html.slim:45 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/regular_events/_regular_event.html.slim:60 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/reports/_report_body.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/reports/show.html.slim:72 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/shared/_modal_learning_completion.html.slim:2 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/survey_questions/_form.html.slim:50 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/survey_questions/_form.html.slim:51 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/survey_questions/_form.html.slim:52 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/talks/_nav.html.slim:6 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/talks/show.html.slim:86 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:101 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:103 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:105 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:11 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:114 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:114 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:124 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:16 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:16 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:17 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:17 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:18 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:18 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:19 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:19 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:23 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:24 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:28 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:28 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:3 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:55 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:56 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:57 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:58 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:65 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:74 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:76 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:78 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:86 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:86 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_form.html.slim:99 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_inactive_users.html.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_password_change.html.slim:15 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/_profile.html.slim:36 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/form/_description.html.slim:11 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/practices/_completed_practices.html.slim:5 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/users/practices/_completed_practices.html.slim:7 [W] RuboCop: Style/HashSyntax: Omit the hash value.
app/views/welcome/_administrator.slim:12 [W] RuboCop: Style/HashSyntax: Omit the hash value.

### CODE REVIEW

**Comment by 88-99** [comment]
@Kassy0220

お疲れ様です。本日ふりかえりミーティングに初参加しましたdawaと申します。
ご都合のよろしいタイミングでレビューをしていただけないでしょうか？🙏

File: Gemfile.lock [hunk 1]
```diff
@@ -289,8 +289,8 @@ GEM
       minitest (>= 5.0.6)
     minitest-retry (0.2.2)
       minitest (>= 5.0)
-    msgpack (1.7.2)
```
Comment (Kassy0220) [inline Gemfile.lock:292]: この変更差分は今回のPRとは関係がないため、このPRには含めない方が良いと思います〜

**Review by Kassy0220 — COMMENTED** [review]
@88-99 
お疲れ様です！
変更差分に関してコメントしていますので、ご確認をお願いいたします🙏

また、PRに関して次のことが気になりました。
1. ブランチ名が`chore/fix-slim-lint-issues-6994`となっていますが、ブランチ名の接頭辞は基本`bug`, `hotfix`, `feature`ではないでしょうか？
[チーム開発の作業手順 \| FBC](https://bootcamp.fjord.jp/pages/305)
2. PRのscreenshotの書き方は[こちらのPR](https://github.com/fjordllc/bootcamp/pull/7000)や[こちらのPR](https://github.com/fjordllc/bootcamp/pull/7013)を参考にされたと思いますが、どのファイルが修正されて警告が無くなったのかとても分かりづらいと感じました。
[2525nicoleさんが作成されたPR](https://github.com/fjordllc/bootcamp/pull/7049)ではとても丁寧に概要が書かれているため、こちらを参考にされると変更箇所が一目で分かるのでありがたいなと思います。

File: Gemfile.lock (hunk 1 above)
Comment (88-99) [reply Gemfile.lock:292]: 早速ご確認いただき、誠にありがとうございます。

> 1. ブランチ名がchore/fix-slim-lint-issues-6994となっていますが、ブランチ名の接頭辞は基本bug, hotfix, featureではないでしょうか？
チーム開発の作業手順 | FBC

質問タイムにて町田さんから「これはbugではなくchore が正しいですよ」というのを教えていただいたので、そのように書きました。
調べたところ chore とは 雑用という意味があり、イシューの横に雑事と記載があるので、その意味で使われるのだと思います。

[Image from Gyazo](https://gyazo.com/674d919c955b4d1b9a71b2c687315151)

> 2. PRのscreenshotの書き方はこちらのPRやこちらのPRを参考にされたと思いますが、どのファイルが修正されて警告が無くなったのかとても分かりづらいと感じました。

ご指摘いただき、ありがとうございます。以後、読む側がわかりやすいよう心がけたいと思います。

#### 警告
 [W] RuboCop: Style/HashSyntax: Omit the hash value.

#### 修正箇所
app/views/admin/users/_nav.html.slim:6

app/views/admin/users/_sort_column.html.slim:3
app/views/admin/users/_sort_column.html.slim:3
app/views/admin/users/_sort_column.html.slim:3

app/views/application/_global_nav.slim:14

app/views/application/_page_tabs.html.slim:8
app/views/application/_page_tabs.html.slim:11
app/views/application/_page_tabs.html.slim:13 


> この変更差分は今回のPRとは関係がないため、このPRには含めない方が良いと思います〜

ありがとうございます。
こちらの手順としては、リモートのプッシュを取り消し、差分を含めないコミットを再プッシュするのだろうか？と考えているのですが、方法がすぐはわからないので調べて修正後にまたご連絡させていただきたいと思います。

Comment (Kassy0220) [reply Gemfile.lock:292]: > 質問タイムにて町田さんから「これはbugではなくchore が正しいですよ」というのを教えていただいたので、そのように書きました。

質問タイムで町田さんに相談されていたのですね！
それならばブランチ名はそのままで大丈夫です👍

> こちらの手順としては、リモートのプッシュを取り消し、差分を含めないコミットを再プッシュするのだろうか？と考えているのですが、方法がすぐはわからないので調べて修正後にまたご連絡させていただきたいと思います。

Gemfile.lock を変更したコミットをローカルで削除し、`push --force-with-lease`で強制プッシュすればうまくいくかなと思います。
dawaさんの方でも調べてみてください〜

Comment (88-99) [reply Gemfile.lock]: @Kassy0220 

方法を教えていただき、ありがとうございます！わからなかったので助かりました。
手順は、`git rebase -i `で該当のコミットを選んで削除後、`push --force-with-lease origin HEAD`しました。
いかがでしょうか？

Comment (Kassy0220) [reply Gemfile.lock]: ご対応ありがとうございます、大丈夫だと思います！

**Review by Kassy0220 — APPROVED** [review]
@88-99 
確認しました！
LGTMですのでApproveさせていただきます〜！🙌🙌

File: Gemfile.lock (hunk 1 above)
Comment (88-99) [reply Gemfile.lock]: レビューいただき、ありがとうございました🙇‍♂️
順番が前後してしまい申し訳ないのですが、ローカルテストで通らないものがいくつかあり対応方法などを調べております。

**Comment by 88-99** [comment]
@komagata 

Approve をいただきましたので、レビューをお願いできますでしょうか？

**Comment by 88-99** [comment]
@komagata 
Approve いただきましたので、レビューをお願いできますでしょうか？

**Comment by komagata** [comment]
@88-99 PRのタイトルは日本語がいいと思いもます。
関連するIssueもシステム的に連携できるのでタイトルに入れないほうがいいです（見る人が混乱してしまうので）

他の人のPRを色々と参考にしてみてください〜。

**Comment by 88-99** [comment]
@komagata 

> PRのタイトルは日本語がいいと思いもます。 関連するIssueもシステム的に連携できるのでタイトルに入れないほうがいいです（見る人が混乱してしまうので）

ご確認とご指摘ありがとうございます！
PRタイトル変更しました。
> Hashの値を省略し警告を解消 #7058

お願いいたします。

**Pull request merged**